relay-server/src/actors/envelopes.rs

```diff
@@ -544,14 +544,14 @@ fn extract_session_metrics(session: &SessionUpdate, target: &mut Vec<Metric>) {
         }
     }
 
-    if session.status.is_terminal() {
+    if matches!(session.status, SessionStatus::Exited) {
         if let Some(duration) = session.duration {
             target.push(Metric {
                 name: "session.duration".to_owned(),
                 unit: MetricUnit::Duration(DurationPrecision::Second),
                 value: MetricValue::Distribution(duration),
                 timestamp,
-                tags: with_tag(&tags, "session.status", session.status),
```

Comment:
I don’t really know enough of metrics, but are we still able to query/group-by in this case? It should not matter cardinality-wise, right?

Reply:
we only emit for a single session.status so all the tag values will be "exited"?